src/components/existing-request/existing-request-display.vue

```diff
@@ -610,25 +613,13 @@ export default class ExistingRequestDisplay extends Mixins(
     // else do nothing -- errors are handled by newReqModule
   }
 
-  private navigateToPaymentPortal () {
-    const { id, token, nrId, action } = this.pendingPayment
-    sessionStorage.setItem('paymentInProgress', 'true')
-    sessionStorage.setItem('paymentId', id)
-    sessionStorage.setItem('paymentToken', token)
-    sessionStorage.setItem('nrId', nrId)
-    sessionStorage.setItem('paymentAction', action)
-    const baseUrl = getBaseUrl()
-    const redirectUrl = encodeURIComponent(`${baseUrl}/nr/${nrId}/?paymentId=${id}`)
-    this.redirectToPaymentPortal(id, token, redirectUrl)
-  }
-
   private async refresh (event) {
     this.$root.$emit('showSpinner', true)
     this.refreshCount += 1
     try {
-      const resp = await NamexServices.getNameRequest(true) as any // *** TODO use a real type here
+      const resp = await NamexServices.getNameRequest(true)
       this.$root.$emit('showSpinner', false)
-      if (resp?.furnished === 'Y') {
+      if (resp?.furnished === Furnished.Y) {
```

Comment:
What is `Furnished`?

Reply:
I'm not sure of the exact business meaning of that. Linda or Sienna would know.